confirmation/migrations/0002_realmcreationkey.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-from django.db import models, migrations
 import django.utils.timezone
+from django.db import migrations, models
```

Comment:
This is a little puzzling -- `django.db` alphabetizes before `django.utils`. Why does isort put it afterward?

Reply:
If I had to guess, the `from` via `import` is what's going on, but it's a bit weird.  @rht can you investigate this?